trino-aws-proxy-spi/src/main/java/io/trino/aws/proxy/spi/plugin/config/PluginOptionalIdentifierConfig.java

```diff
@@ -11,16 +11,11 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package io.trino.aws.proxy.spi;
+package io.trino.aws.proxy.spi.plugin.config;
 
-import com.google.inject.Module;
+import java.util.Optional;
 
-public interface TrinoAwsProxyServerPlugin
+public interface PluginOptionalIdentifierConfig
```

Comment:
nit: for me, `Optional` in the name is redundant. It's implied by the method return type.